<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tGisAppClass</class>
 <widget class="QMainWindow" name="tGisAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tGis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QMapWidget" name="mapWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="zoomInAction"/>
   <addaction name="zoomOutAction"/>
   <addaction name="zoomFreeAction"/>
   <addaction name="zoomRectAction"/>
   <addaction name="panAction"/>
   <addaction name="entireMapAction"/>
   <addaction name="refreshMapAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>图层</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLayerWidget" name="layerWidget">
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>数据源</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QDataSourceWidget" name="dataSourceWidget">
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="zoomInAction">
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/Zoomin.png</normaloff>:/tGisApp/Resources/Zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/Zoomout.png</normaloff>:/tGisApp/Resources/Zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="zoomFreeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/ZoomFree.png</normaloff>:/tGisApp/Resources/ZoomFree.png</iconset>
   </property>
   <property name="text">
    <string>自由缩放</string>
   </property>
   <property name="toolTip">
    <string>自由缩放</string>
   </property>
  </action>
  <action name="zoomRectAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/ZoomRect.png</normaloff>:/tGisApp/Resources/ZoomRect.png</iconset>
   </property>
   <property name="text">
    <string>拉框放大</string>
   </property>
   <property name="toolTip">
    <string>拉框放大</string>
   </property>
  </action>
  <action name="panAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/Pan.png</normaloff>:/tGisApp/Resources/Pan.png</iconset>
   </property>
   <property name="text">
    <string>自由移动</string>
   </property>
   <property name="toolTip">
    <string>自由移动</string>
   </property>
  </action>
  <action name="entireMapAction">
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/Entire.png</normaloff>:/tGisApp/Resources/Entire.png</iconset>
   </property>
   <property name="text">
    <string>全图</string>
   </property>
   <property name="toolTip">
    <string>全图</string>
   </property>
  </action>
  <action name="refreshMapAction">
   <property name="icon">
    <iconset resource="tGisApp.qrc">
     <normaloff>:/tGisApp/Resources/RefreshMap.png</normaloff>:/tGisApp/Resources/RefreshMap.png</iconset>
   </property>
   <property name="text">
    <string>刷新地图</string>
   </property>
   <property name="toolTip">
    <string>刷新地图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDataSourceWidget</class>
   <extends>QTreeView</extends>
   <header>qdatasourcewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QMapWidget</class>
   <extends>QWidget</extends>
   <header>qmapwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLayerWidget</class>
   <extends>QTreeView</extends>
   <header>qlayerwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tGisApp.qrc"/>
 </resources>
 <connections/>
</ui>
